src/lib/style/components/_query-filter.scss

```diff
@@ -1,10 +1,15 @@
 @use '../abstracts/variables' as SRC;
 
 .QueryFilter {
-  margin: 0rem 1rem 1rem 0;
-  border: 1px solid SRC.$border-color-gray;
-  padding: 1rem;
-
+  &__title {
+    font-weight: bold;
+  }
+  padding: 2rem;
+  background-color: adjust-color(
```

Comment:
very light themed primary action color

Reply:
Ah, I use `background-color: rgba(SRC.$primary-action-color-500, 0.1);`. But this should also work!